en/components/grids_templates/row_editing.md

```diff
@@ -299,6 +299,199 @@ If you want the buttons to be part of the keyboard navigation, then each on of t
+);
+```
+
+Our custom grid and input-group themes will take care of the editing parts in the grid.
+Now, all that's left is to define two more themes to style the row-editing overlay. The overlay itself is composed of a banner (the background-part) and the custom text and buttons we passed.
+To style the banner, we'll create a custom banner theme and scope it to the overlay container:
+```scss
+$banner-theme: igx-banner-theme(
+    $banner-background: igx-color($default-palette, "primary", 100),
+    $banner-message-color: igx-contrast($default-palette, "primary", 100),
+    $banner-border-color: igx-color($default-palette, "second", 800)
+);
+```
+
+For the buttons, we'll need to create two themes - since we want the buttons to be different from one another and to have either a red or green highlight (for `dismiss` and `done`, resp.):
+```scss
+$button-theme-success: igx-button-theme(
+    $icon-color: igx-contrast($my-success-error-palette, "primary", 200),
+    $icon-hover-color: igx-contrast($my-success-error-palette, "primary", 600),
+    $icon-focus-color: igx-contrast($my-success-error-palette, "primary", 300),
+    $icon-background: igx-color($my-success-error-palette, "primary", 200),
+    $icon-hover-background: igx-color($my-success-error-palette, "primary", 600),
+    $icon-focus-background: igx-color($my-success-error-palette, "primary", 300)
+);
+$button-theme-failure: igx-button-theme(
+    $icon-color: igx-contrast($my-success-error-palette, "secondary", 200),
+    $icon-hover-color: igx-contrast($my-success-error-palette, "secondary", 600),
+    $icon-focus-color: igx-contrast($my-success-error-palette, "secondary", 300),
+    $icon-background: igx-color($my-success-error-palette, "secondary", 200),
+    $icon-hover-background: igx-color($my-success-error-palette, "secondary", 600),
+    $icon-focus-background: igx-color($my-success-error-palette, "secondary", 300)
+);
+```
+
+To define the two themes, we use the custom palette we created earlier, since the `igx-palette` function conveniently creates color shades for us to use!
+
+#### Applying
+
+All that's left is to properly scope our newly created themes to our component.
+For the buttons, we can use our class selector that we added earlier, in the [templates](#passing-custom-templates) section
+
+```scss
+.custom-failure {
+    @include igx-button($button-theme-failure);
+  }
+
+.custom-success {
+    @include igx-button($button-theme-success);
+  }
+```
+
+As for input, grid and banner themes, we pass them under the context of our sample component:
+```scss
+.sample-wrapper {
+    @include igx-grid($custom-grid);
+    @include igx-input-group($input-group);
+    @include igx-banner($banner-theme);
+}
+```
+>[!NOTE]
+ >If the component is using an [`Emulated ViewEncapsulation`](../themes/component-themes.md#view-encapsulation), it is necessary to penetrate this encapsulation using `::ng-deep`:
+ ```scss
+    ::ng-deep {
+        .sample-wrapper {
```

Comment:
nest `::ng-deep` inside the `.sample-wrapper` instead, otherwise other elements with this class might be affected.